### DISCUSSION TITLE
Integration Suggestion for Whisper and Pyannote Speaker Diarization Model

### DISCUSSION THREAD

**Discussion by ns1bdwn** [Purfview/whisper-standalone-win#241] [Q&A] [answered]
Hi Purview,

Thank you so much for your excellent work on the project—it's truly appreciated.

I wanted to discuss the potential of integrating the Whisper model with the Pyannote Speaker Diarization model. You can find more details about the Pyannote model here: https://huggingface.co/pyannote/speaker-diarization.

I've also come across this existing solution: https://huggingface.co/spaces/sanchit-gandhi/whisper-jax-diarization. It's quite impressive, but given your expertise, I'm confident you could enhance its speed and efficiency.

Looking forward to your thoughts on this possibility.

**Comment by Purfview** [comment, marked as answer]
Dupe of:
https://github.com/Purfview/whisper-standalone-win/discussions/120
https://github.com/Purfview/whisper-standalone-win/discussions/234
